src/index.tsx

```diff
@@ -51,7 +51,7 @@ ReactDOM.render(
     // Globally disable ripples
     ripple={false}
   >
-    <ThemeProvider options={{ background, primary }}>
+    <ThemeProvider options={{ background, primary, secondary: '#F5F5F5' }}>
```

Comment:
Move this also into `colors`?

Reply:
Also, thoughts on saving secondary for the `#476282` color? It's use in quite a few places like overview text and banners in RTDB.

Reply:
Ack, good call... moved to colors and also isolated this surface within panel header